### ISSUE TITLE
Get translated headings for Features & Analysis, Top Stories components

### ISSUE THREAD

**Issue by mykeels** [bbc/simorgh#5794]
**Is your feature request related to a problem? Please describe.**
The https://github.com/bbc/simorgh/pull/5751 PR currently has no translations for the "Features & Analysis" header. Soon, we'll be working on Top Stories. We need to ensure that translations exist for these headings.

**Describe the solution you'd like**
- Get translated headings for Features & Analysis, Top Stories components.
- In the a11y-swarm, check the NVDA Firefox issues highlighted in [this PR](https://github.com/bbc/simorgh/pull/5751).
- Investigate the effect of removing the summary in the F&A component, on other OJ components.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by lizcameron** [comment]
Hi @mykeels 

The headings for 'Features and Analysis' and 'Top Stories' come from the front page as well. The data for the components comes from a CPS group  on the front page (the CPS group is different for each onward journey type - see #5700 for Top Stories and #5701 for Features and Analysis).

Each CPS group has a group name, which is defined as `group.strapline.name`. This is what becomes the header for the component.

E.g. for the Thai service, the front page data contains the following:

```
strapline: {
name: "ข่าวเด่น"
},
semanticGroupName: "Top story"
```
See: https://ares-broker.api.bbci.co.uk/api/index/thai/front_page
So the Top Stories onward journey on the STY page will take the content from the CPS group 'Top story', which has the name: 'ข่าวเด่น'. So this is what gets pulled through to the STY page as the heading.

Let me know if you have any questions.

**Comment by mykeels** [comment]
Blocked by https://github.com/bbc/simorgh/issues/5737

**Comment by lizcameron** [comment]
For re-refinement: clarify the following a11y issue to be resolved for the Feature and Analysis MVP.
- In the a11y-swarm, check the NVDA Firefox issues highlighted in this PR.

OR revisit the issue when we complete a11y testing for Features and Top Stories later on.

Notes:
- Translations will be part of the CPS Group data we get from AReS
- Summary is currently displayed at the medium breakpoint only and this is correct, intended behaviour (insofar as it matches the behaviour of the component elsewhere) - this needs to be discussed with the Home and Media team to decide what the desired behaviour should be to keep consistency across pages, and any issues to improve this will be raised separately. Note that summary will still be read out by some AT even if it's not displayed.

**Comment by lizcameron** [comment]
Closed as the points mentioned here are captured in other issues:
- The NVDA Firefox issues highlighted in [this PR](https://github.com/bbc/simorgh/pull/5751) - can be picked up in the full a11y swarm for these components https://github.com/bbc/simorgh/issues/6088
- Translations - will be part of the AReS data, see https://github.com/bbc/simorgh/issues/5765
- Summary - behaving as expected. Bug around summary display at medium breakpoint captured in Jira backlog and to be discussed with UX, Home and Media teams. If needed, a separate issue can be raised.
